src/test/java/tests/ZipCodesTest.java

```diff
@@ -27,8 +30,11 @@ public void postZipCodesTest() {
     }
 
     @Test
+    @Step("POST Zipcode duplicate in list, Request payload: {payload}")
+    @Issue("PlaceholderBugID")
```

Comment:
Usually, the Issue annotation contains the corresponding ticket for the bug in the Jira. Ex.: `@Issue("SYS-2568")` In your case just try to put there smth like short description